{"name": "test_mark_routine", "status": "broken", "statusDetails": {"message": "TypeError: 'int' object is not callable", "trace": "login_token = 'eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiI1MiIsImV4cCI6MTcxNzk0NzkzMSwicm9sZXMiOlsiVVNFUiJdLCJlbWFpbCI6InNoaW5hODIxNEBmdGhjYXBpdGFsLmNvbSJ9.p2AhGHEjbx7sOT5ePNjdGbb1eEQk6lAfflGNvRX-KZg'\ncreate_routine = 491\n\n    @allure.feature('Routine')\n    @allure.story('Mark routine')\n    def test_mark_routine(login_token, create_routine):\n        with allure.step('Create Routine'):\n            headers = {\n                \"Authorization\": f\"Bearer {login_token}\",\n                \"Accept\": \"*/*\",\n            }\n>           new_routine = create_routine()\nE           TypeError: 'int' object is not callable\n\ntests/test_routine.py:108: TypeError"}, "steps": [{"name": "Create Routine", "status": "broken", "statusDetails": {"message": "TypeError: 'int' object is not callable\n", "trace": "  File \"/Users/alya/shit/tmm/tests/test_routine.py\", line 108, in test_mark_routine\n    new_routine = create_routine()\n                  ^^^^^^^^^^^^^^^^\n"}, "start": 1717947332389, "stop": 1717947332389}], "start": 1717947332389, "stop": 1717947332389, "uuid": "324b3cda-ec13-4d0b-bdbf-bcb6e03ad07a", "historyId": "e45539616ee0a6837b10d6ddb5630e7a", "testCaseId": "e45539616ee0a6837b10d6ddb5630e7a", "fullName": "tests.test_routine#test_mark_routine", "labels": [{"name": "feature", "value": "Routine"}, {"name": "story", "value": "Mark routine"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_routine"}, {"name": "host", "value": "Air-makbuk.Dlink"}, {"name": "thread", "value": "51159-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_routine"}]}